app/submitter/convert_payload_0_0_3.py

```diff
@@ -85,23 +95,29 @@ def add_list_collector_answers(
     list_item_ids = list_store[list_name].items
 
     for list_item_id in list_item_ids:
-        for answer_id in answers_ids_in_add_block:
-            answer = answer_store.get_answer(answer_id, list_item_id)
-            if answer:
-                answers_payload.add_or_update(answer)
+        if answers_ids_in_add_block:
```

Comment:
This should be outside the loop. We can just return early if it doesn't exist.

Reply:
Moved it outside the loop, thanks!